lib/Conversion/TritonGPUToLLVM/ScanOpToLLVM.cpp

```diff
@@ -488,8 +498,19 @@ ScanOpConversion::emitFastScan(triton::ScanOp op, triton::ScanOpAdaptor adaptor,
   laneMask = (layout.getInDimSize(kLane) - 1) & ~laneMask;
   laneId = b.and_(laneId, b.i32_val(laneMask));
 
-  auto [laneIdAxis, warpIdAxis, flatIdParallel] =
-      getDelinearizedIds(rewriter, helper, laneId, warpId);
+  auto [laneIdAxis, warpIdAxis, flatIdParallel, cctaIdAxis, cctaIdParallel] =
+      getDelinearizedIds(rewriter, helper, laneId, warpId, cctaId);
```

Comment:
I don't think the math should consider `cctaId` if the scan is not happening cross ctas